tests/RA_UnitTestHelpers.h

```diff
@@ -26,6 +26,12 @@ namespace CppUnitTestFramework {
 
 #pragma warning(push)
 #pragma warning(disable : 4505) // unreferenced inline functions, they are referenced. Must be a bug.
+template<>
+std::wstring ToString<std::chrono::milliseconds>(const std::chrono::milliseconds& t)
+{
+    return std::to_wstring(t.count()) + L"ms";
```

Comment:
The template arguments don't have to be there for specializations.
You could write it like:
```
template<>
std::wstring ToString(const std::chrono::milliseconds& t)
```

I also think you should use one of the methods in `StringUtils` instead of using `+`

Reply:
It's fine